MANIFEST.in

```diff
@@ -1,12 +1,12 @@
 include AUTHORS
 include LICENSE
-include DESCRIPTION
 include CHANGELOG.txt
 include README.md
-include requirements.txt
 graft hijack
 global-exclude *.orig *.pyc *.log *.swp
 prune hijack/tests/coverage
 prune hijack/.ropeproject
+prune .github
+exclude .gitignore
 recursive-include hijack/templates *
 recursive-include hijack/locale *
```

Comment:
`MANIFEST.in` looks scary big. I'm not sure, but I think it could be much shorter.

Reply:
I will do a second sweep, try to see if we can't condense this a little more. I always inspect the dist packages to verify we don't miss anything. I will ping you once I am done.

Reply:
Good call, the `graft` statement caused all defaults to be overridden. Now it's a lot simpler.

Reply:
Nice!